pkg/compiler/codegen.go

```diff
@@ -1242,6 +1249,24 @@ func (c *codegen) processDefers() {
 	}
 }
 
+// emitExplicitConvert handles `someType(someValue)` conversions between string/[]byte.
+// Rules for conversion:
+// 1. interop.* types are converted to ByteArray if not already.
+// 2. Otherwise convert between ByteArray/Buffer.
+// 3. Rules for types which are not string/[]byte should already
+//    be enforced by go parser.
+func (c *codegen) emitExplicitConvert(from, to types.Type) {
+	if isInteropPath(to.String()) {
```

Comment:
Don't we also have structures there like `Block` or `Transaction`?

Reply:
Fixed.